environments/openshift/obs.jsonnet

```diff
@@ -486,6 +487,30 @@ local cqf = (import '../../components/cortex-query-frontend.libsonnet');
     },
   },
 
+  local upComponent = up + up.withResources {
+    config+:: {
+      local cfg = self,
+      name: obs.config.name + '-' + cfg.commonLabels['app.kubernetes.io/name'],
+      namespace: obs.config.namespace,
+      readEndpoint: 'http://%s.%s.svc/api/v1/query' % [obs.queryCache.service.metadata.name, obs.queryCache.service.metadata.namespace],
+      version: 'master-2020-03-25-6d4f944',
+      image: 'quay.io/observatorium/up:' + cfg.version,
+      queryConfig: (import 'queries.libsonnet'),
+      resources: {
+        requests: {
+          cpu: '5m',
+          memory: '10Mi',
+        },
+        limits: {
+          cpu: '20m',
```

Comment:
This can really affect test runs from client side. I would give 1CPU, we are not short on CPU I think

Reply:
Right now all this thing does is to an HTTP request.